src/app/integrations/qbo/qbo-main/qbo-dashboard/qbo-dashboard.component.spec.ts

```
@@ -1,23 +1,275 @@
+    accountingExportService = TestBed.inject(AccountingExportService) as jasmine.SpyObj<AccountingExportService>;
+    dashboardService = TestBed.inject(DashboardService) as jasmine.SpyObj<DashboardService>;
+    workspaceService = TestBed.inject(WorkspaceService) as jasmine.SpyObj<WorkspaceService>;
+    qboExportSettingsService = TestBed.inject(QboExportSettingsService) as jasmine.SpyObj<QboExportSettingsService>;
+    importSettingService = TestBed.inject(QboImportSettingsService) as jasmine.SpyObj<QboImportSettingsService>;
+  });
 
+  beforeEach(() => {
     fixture = TestBed.createComponent(QboDashboardComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
   });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)